apps/prairielearn/src/pages/instructorQuestionSettings/instructorQuestionSettings.html.ts

```diff
@@ -228,6 +228,66 @@ export function InstructorQuestionSettings({
                   : ''}
               </table>
             </div>
+            <div class="form-group">
+              <label for="grading_method">Grading Method</label>
+              <select
+                class="form-select"
+                id="grading_method"
+                name="grading_method"
+                value="${resLocals.question.title}"
+                ${canEdit ? '' : 'disabled'}
+              >
+                <option
+                  value="Internal"
+                  ${resLocals.question.grading_method === 'Internal' ? 'selected' : ''}
+                >
+                  Internal
+                </option>
+                <option
+                  value="External"
+                  ${resLocals.question.grading_method === 'External' ? 'selected' : ''}
+                >
+                  External
+                </option>
+                <option
+                  value="Manual"
+                  ${resLocals.question.grading_method === 'Manual' ? 'selected' : ''}
+                >
+                  Manual
+                </option>
+              </select>
+              <small class="form-text text-muted">
+                The grading method used for this question.
+              </small>
+            </div>
+            <div class="form-group">
+              <input
+                class="form-check-input"
+                type="checkbox"
+                id="single_variant"
+                name="single_variant"
+                ${canEdit ? '' : 'disabled'}
+                ${resLocals.question.single_variant ? 'checked' : ''}
+              />
+              <label class="form-check-label" for="single_variant">Single Variant</label>
+              <small class="form-text text-muted">
+                If checked, the question is not randomized and only generates a single variant.
+              </small>
```

Comment:
Nit: let's use a block-level element here instead of `<span>`. That'll put the explanatory text on a separate line, which IMO is easier to parse.

Here's an example of this from the global admin SSO config page:

https://github.com/PrairieLearn/PrairieLearn/blob/43bcfa9edcd6fbef7936081213cc78d092360b86/apps/prairielearn/src/ee/pages/administratorInstitutionSaml/administratorInstitutionSaml.html.ts#L163-L178

And here's how it looks rendered:

<img width="303" alt="Screenshot 2025-02-04 at 09 32 04" src="https://github.com/user-attachments/assets/5b2a9920-213e-490d-93e5-0afecce12e8c" />

Reply:
Looks much better, thanks!